judge/views/problem.py

```diff
@@ -219,9 +219,64 @@ def get_context_data(self, **kwargs):
                                           context['description'], 'problem')
         context['meta_description'] = self.object.summary or metadata[0]
         context['og_image'] = self.object.og_image or metadata[1]
+
+        context['can_vote'] = self.object.can_vote(user)
+        # The vote this user has already cast on this problem.
+        if context['can_vote']:
+            try:
+                vote = ProblemPointsVote.objects.get(voter=user.profile, problem=self.object)
```

Comment:
This...does not look like the code I posted.

Reply:
Oh I forgot to change that one